dashlive/mpeg/dash/validator/dash_element.py

```diff
@@ -183,6 +192,24 @@ def find_parent(self, name: str) -> Optional["DashElement"]:
             return self.parent.find_parent(name)
         return None
 
+    @classmethod
+    def xpath(cls, xml: ET.ElementBase, path: str) -> list[ET.ElementBase | str]:
+        """
+        Find an element using XPath. In addition to the XPath features of
+        lxml, this function also supports indexing (e.g. Element[2])
```

Comment:
Correct the comment. lxml supports indexing, but doesn't support default namespaces.